README.md

```diff
@@ -74,6 +74,42 @@ jobs:
       - run: npm test
 ```
 
+Operating Systems and Architecture:
+
+You can use any of the [supported operating systems](https://docs.github.com/en/actions/reference/virtual-environments-for-github-hosted-runners). The architecture can be selected using `node-arch`. Values are `x86`, `x64`, `arm64`, `armv6l`, `armv7l`, `ppc64le`, `s390x` (not all of the architectures are available on all platforms).
+```yaml
+jobs:
+  build:
```

Comment:
Could you please add a simple use case for the architecture input?

Architecture:
```
steps:
- uses: actions/checkout@v2
- uses: actions/setup-node@v1
  with:
    node-version: '12'
    architecture: 'x64' # optional, x64 or x86. If not specified, x64 will be used by default
- run: npm install
- run: npm test
```

Reply:
Fixed in https://github.com/actions/setup-node/pull/194/commits/5766bdce1ce748ecf462ec1c6007d9842d53853f

Reply:
Hi folks, thanks for the PR. A bit late to say this, but I only just noticed it:

>    `architecture: [...] optional, x64 or x86. If not specified, x64 will be used by default`

There are self-hosted Linux ARM (`armv7l`) and ARM64 (`arm64`) runners. So this comment is a bit misleading for those non-x64/non-x86 runners.

I suppose the comment could be:

`architecture: 'x64' # optional. If not specifiied, this will default to your runner's architecture`